app/models/bacon.rb

```diff
@@ -1,3 +1,3 @@
-class Bacon < ActiveRecord::Base
-  
+class Bacon < ApplicationRecord
```

Comment:
Upgrade to Rails 5
http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#active-record-models-now-inherit-from-applicationrecord-by-default